lib/find_process.js

```diff
@@ -123,7 +123,8 @@ const finders = {
       const cmd = 'WMIC path win32_process get Name,Processid,ParentProcessId,Commandline,ExecutablePath'
       const lines = []
 
-      const proc = utils.spawn('cmd', ['/c', cmd], { detached: false, windowsHide: true })
+      const spawnCmd = process.env.path && process.env.path.includes('C:\\Windows\\System32') ? 'cmd' : 'C:\\Windows\\System32\\cmd.exe'
+      const proc = utils.spawn(spawnCmd, ['/c', cmd], { detached: false, windowsHide: true })
```

Comment:
path on windows is case-insensitive, I prefer using regex pattern match instead of `path.includes`.